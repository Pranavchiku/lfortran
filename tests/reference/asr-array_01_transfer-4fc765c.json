{
    "basename": "asr-array_01_transfer-4fc765c",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/array_01_transfer.f90",
    "infile_hash": "920caa9e18706045e10e552111ebbdee612fb11bdc829a77fcfa71ca",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-array_01_transfer-4fc765c.stdout",
    "stdout_hash": "248eb4dca202e05afde7b689610661ae59444e17cc27466eb5b9a273",
    "stderr": "asr-array_01_transfer-4fc765c.stderr",
    "stderr_hash": "fa8b2e1ff8f3dae7374bde73665ae7549b1cbba0c42d1966e74d8a19",
    "returncode": 0
}