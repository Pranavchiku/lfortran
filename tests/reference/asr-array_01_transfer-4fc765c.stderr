warning: Implied save feature
 --> tests/../integration_tests/array_01_transfer.f90:2:3
  |
2 |   integer :: m(6), size = 2
  |   ^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
